variables.yml

```
@@ -9,6 +9,7 @@ networks:
       erc20: '0x0000000000000000000000000000000000000800'
       batch: '0x0000000000000000000000000000000000000801'
       call_permit: '0x0000000000000000000000000000000000000802'
+      xcmInterface: '0x0000000000000000000000000000000000000804'
```

```suggestion
      xcm_interface: '0x0000000000000000000000000000000000000804'
```
have to update every where we used this too